### ISSUE TITLE
1.1/dev installation issue

### ISSUE THREAD

**Issue by ChristianGiupponi** [firesale/firesale#145]
I've just donwload the zip from 1.1dev but when I try to install I get:

`An Error Was Encountered
empty_stream_name [add_stream]
`

Logs says:

`ERROR - 2012-10-22 10:03:21 --> empty_stream_name [add_stream]`

**Comment by ChristianGiupponi** [comment]
This problem append only if the langue is not set to english, maybe is just a missing string in the italian language file

**Comment by jHoldroyd** [comment]
I think was related to the last commit I made, I ran another complete diff of the language files and found 4-5 items missing for a few of them (thank god for automation tools!). This may have caused your problems and if you can give it a try and let me know that'd be great.

**Comment by ChristianGiupponi** [comment]
Perfect, now it works!

**Comment by ChristianGiupponi** [comment]
There is the same problem again. I can't install firesale in italian

**Comment by ChristianGiupponi** [comment]
This should fix for italian lang: c2e0603884720078d0d757a8379ea31fcb0cd573

**Comment by jHoldroyd** [comment]
I wrote a script a while ago to diff all of the language files at once, if you think it'd be useful to you I don't mind sending it over. You just stick it in the root of the module you're doing, call it with a browser and it'll run through all languages compared to a defined base language and show all missing items from each of them in one page.

I found it really handy a few weeks ago after you had the issue with installing in Italian then too.
